maestro-utils/src/main/kotlin/Insight.kt

```diff
@@ -1,31 +1,20 @@
 package maestro.utils
 
-object Insights {
+object Insights: InsightsInterface {
```

Comment:
IIUC, this doesn't do anything, other than invoking a bunch of listeners. Why do you need a noop version in that case?

Reply:
The Insights are only used for CLI, the noop is default implementation that we pass for cloud versions and only override for CLI. Earlier the implementations were coupled inside Orchestra and hence the cloud code.